app/models/project/export/info_request.rb

```diff
@@ -0,0 +1,60 @@
+##
+# Export key classification and extracted data for an info request within a
+# given project.
+#
+class Project::Export::InfoRequest
+  extend Forwardable
+
+  attr_reader :project, :info_request
```

Comment:
Should make these [`private`/`protected`](https://www.garethrees.co.uk/2020/06/14/use-protected-attr-readers/#why-not-use-public-accessors).

Reply:
As above I've changed this. Also switched the class to inherit from SimpleDelegator to remove #info_request completely